thirdparty/vulkan/vk_mem_alloc.h

```diff
@@ -122,16 +122,13 @@ for user-defined purpose without allocating any real GPU memory.
 See documentation chapter: \ref statistics.
 */
 
+#include "drivers/vulkan/godot_vulkan.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-#ifdef USE_VOLK
-    #include <volk.h>
-#else
-    #include <vulkan/vulkan.h>
-#endif
+
```

Comment:
We're unintentionally adding a stray line here, would be good to remove it (also in the .patch) to keep our changes minimal.

What's the rationale from moving the include outside the `extern "C"` scope though? Does it only work for C++ code?

Reply:
The original error from Clang was that it was no longer allowed to include stdint.h in an extern "C" {} block. That is why the include was moved.

Reply:
Pushed the update with the removal of the extra line.